Mage/src/main/java/mage/abilities/dynamicvalue/common/PermanentsOnBattlefieldCount.java

```diff
@@ -27,6 +28,7 @@ public PermanentsOnBattlefieldCount(FilterPermanent filter) {
      * @param multiplier
      */
     public PermanentsOnBattlefieldCount(FilterPermanent filter, Integer multiplier) {
+        // TODO: this doesn't need a multiplier. use Multipliedvalue
```

Comment:
106 usages.

Note that the reason it's an "Integer" rather than an "int" is that it can be null, and that was used to distinguish phrasing options

Reply:
Right, but those phrasing options will be covered in the new scheme. Using the existence of a multiplier on a value that otherwise uses it in the same way as `MultipliedValue` (if it exists) to generate phrasing is a pretty arbitrary choice, no? At best, it allows people to customize the phrasing of the value at construction, which is a pattern I'm replacing in this rework--phrasing will belong to effects.

> 106 usages.

Is this you saying that you don't think this is necessary to change? Certainly this won't be a change for this PR, this would be further down the road, probably in its own PR tied to #12759

Reply:
Sorry for being unclear - wasn't trying to say you should do anything one way or the other, just noting context in case you weren't aware.

Reply:
No worries lol